sqlite_query.c

```diff
@@ -625,19 +718,19 @@ sqlite_bind_sql_var(Form_pg_attribute att, int attnum, Datum value, sqlite3_stmt
 			}
 		case UUIDOID:
 			{
-				bool		uuid_as_blob = false;
+				int	sqlite_aff = SQLITE_NULL;
 				if (relid)
```

Comment:
Please add a blank line.

Reply:
Added, please verify.

Reply:
Confirmed. Thanks. Please help me to resolve this comment.